src/govuk/i18n.mjs

```diff
@@ -1,3 +1,18 @@
+/**
+ * Associates translated messages to plural type they correspond to.
+ *
+ * Allows to group pluralised messages under a single key when passing
+ * translations to a component's constructor
```

Comment:
I feel like we're a bit inconsistent with some of the language we're using in comments and docs.

Wondering if it's worth us trying to put together a bit of a glossary of terms for internal use? Try and define what we mean when we say things like 'plural forms', 'plural rules' and 'plural types' – if they are even separate things?